go.mod

```diff
@@ -156,11 +156,9 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	// Use the cosmos modified protobufs
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	// Fix rocksdb for high traffic situations
-	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
```

Comment:
got an error with this 
```
go: github.com/cosmos/gorocksdb@v1.2.0 used for two different module paths (github.com/cosmos/gorocksdb and github.com/tecbot/gorocksdb)
```
i think is this related to ethermint update. i believe we add this as a part of ibc ([pr](https://github.com/Kava-Labs/kava/pull/1138)). are we fine to remove this?

Reply:
Yeah, fine to remove